tests/apps/flask_app/pubsub.py

```diff
@@ -0,0 +1,100 @@
+import os
+import json
+import logging
+
+import instana
+
+from flask import Flask, request
+from google.auth import jwt
+from google.cloud import pubsub_v1
+
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+app.debug = True
+app.use_reloader = True
+
+# Use PubSub Emulator exposed at :8432 for local testing
+# os.environ["PUBSUB_EMULATOR_HOST"] = "localhost:8432"
+
+SERVICE_FILE = os.path.join(os.path.dirname(__file__), 'service-account-info.json')
```

Comment:
A minor note, but would it be more handy to read the credentials path from env instead of hardcoding it?

Reply:
Yes, there is a better way to do it. Refactored the whole access mechanism and places everything inside a new test app folder - check the latest changes